src/ert/gui/model/snapshot.py

```diff
@@ -491,7 +488,7 @@ def index(
 
         parent_item = self.root if not parent.isValid() else parent.internalPointer()
         try:
-            child_item = parent_item.children_list[row]
```

Comment:
This is a clean revert when the concept of children_list was introduced in 7c5022e2f1e1f09282cc98729f0d1938131a5704